src/language/dafnyLanguageClient.ts

```diff
@@ -51,6 +52,21 @@ function getMarkGhostStatementsArgument(): string {
   return `--ghost:markStatements=${Configuration.get<string>(ConfigurationConstants.LanguageServer.MarkGhostStatements)}`;
 }
 
+function getDafnyPluginsArgument(): string {
+  let plugins = Configuration.get<string[]>(ConfigurationConstants.LanguageServer.DafnyPlugins);
+  if(plugins === null) {
+    return '';
+  }
+  plugins = plugins.filter(x => x !== null && x !== '');
+  const result = [];
+  for(const i in plugins) {
+    const possiblyEscaped = plugins[i].includes(' ') && !plugins[i].startsWith('"')
+      ? '"' + plugins[i].replace(/\\/g, '\\\\').replace(/"/g, '\\"') + '"' : plugins[i];
+    result.push(`--dafny:plugins:${i}=${possiblyEscaped}`);
+  }
+  return result.join(' ');
+}
+
```

Comment:
If there are command line arguments, they should be escaped before being concatenated, or better yet we should use an API that takes a list of arguments, not a string.

Reply:
And in fact this is what we are doing, so this function should return a `string[]`, not a string (and in the caller you can use `...getDafnyPluginsArguments()`